spec/controllers/facility_accounts_controller_spec.rb

```diff
@@ -2,17 +2,19 @@
 require "controller_spec_helper"
 
 RSpec.describe FacilityAccountsController, feature_setting: { edit_accounts: true } do
-  let(:facility) { @authable }
+  let(:facility) { FactoryGirl.create(:facility) }
+  let(:account) { create_nufs_account_with_owner }
+  let(:admin) { @admin }
```

Comment:
Totally fair question. It gets created by `before(:all) { create_users }`. There's obviously further I could have gone with cleaning up the specs, but creating a few `let`s is where I stopped.